app/frontend/stylesheets/pages/_index.scss

```diff
@@ -1,3 +1,4 @@
 @import "partials/calendar";
 @import "partials/membership";
 @import "home";
+@import "partials/footer";
```

Comment:
Can you move that up between the calendar and membership please

Reply:
done